core/templates/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -550,10 +555,13 @@ angular.module('oppia').directive('questionPlayer', [
               )
             );
 
-            $rootScope.$on('questionSessionCompleted', function(event, result) {
-              $location.hash(HASH_PARAM +
-                encodeURIComponent(JSON.stringify(result)));
-            });
+            ctrl.directiveSubscriptions.add(
```

Comment:
This is not 'unsubscribed' in this file?

Reply:
It is: https://github.com/oppia/oppia/pull/10255/files#diff-cff9dd4f894c11e2634434695a00c69eR595, right?

Reply:
Ah ok!